package.json

```diff
@@ -405,4 +405,4 @@
     "stx",
     "web3"
   ]
-}
+}
```

Comment:
Editorconfig plugin should prevent removing the last line like this, I think also prettier?